src/lib/components/CommunitySelectionModal/CommunitySelectionRSKApp.js

```diff
@@ -0,0 +1,168 @@
+// This file is part of React-Invenio-Deposit
```

Comment:
This file looks like a duplication of [CommunitySelectionSearch](https://github.com/inveniosoftware/react-invenio-deposit/blob/2069bbdf2bc6ddcc65f99619fdfceb39c1c6db33/src/lib/components/CommunitySelectionModal/CommunitySelectionSearch.js). Why is this needed?

Reply:
my bad, it is the same component but i changed the name, it is rebased properly now.